cscs-checks/microbenchmarks/latency/latency.py

```diff
@@ -10,7 +10,8 @@ def __init__(self):
         self.sourcepath = 'latency.cpp'
         self.build_system = 'SingleSource'
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'ault:intel', 'ault:amdvega', 'tave:compute']
+                              'ault:intel', 'ault:amdvega', 'tave:compute',
+                              'tiger:gpu']
```

Comment:
I don't think this test is meaningful for Tiger. I don't think the `craype` tag is needed either.